packages/utils/odsp-doclib-utils/src/resolveWithDomainMove.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IRequest } from "@fluidframework/core-interfaces";
+import { IUrlResolver } from "@fluidframework/driver-definitions";
+import { IOdspResolvedUrl, OdspErrorType } from "@fluidframework/odsp-driver-definitions";
+
+/**
+ * Checks if the error is domain move error.
+ * @param error - error whose type is to be determined.
+ * @returns - True is the error is domain move error.
+ */
+ export function isDomainMoveError(error: any) {
+    if (typeof error === "object" && error !== null
```

Comment:
#nit we can just do return (condition) and drop the `if`

Reply:
done